### PR TITLE
Update Helm Chart for eventing-related properties

### PR DESCRIPTION
**Pull request by corcoja**
Add missing values for latest `lib.eventing` (version `0.2.2`) and NATS Jetstream.

#### Changelog

**Changed**

- Updated `configmap` to use new values from `values.yaml`.

### CODE REVIEW

**Comment by corcoja** [comment]
Update for #43

**Review by tlawrie — DISMISSED** [review]

**Comment by tlawrie** [comment]
Tagging @morarucostel for +2. We will also need a new chart version added to the PR for when its merged in.

**Comment by corcoja** [comment]
@morarucostel, can you please take a look?

**Comment by corcoja** [comment]
@tlawrie @marcusdroy @morarucostel 

This PR goes hand in hand with:

- boomerang-io/flow.service.listener#16
- boomerang-io/flow.service.workflow#126

File: bmrg-flow/README.md [hunk 1]
```diff
@@ -124,13 +126,12 @@ The following table lists the configurable parameters of the chart and their def
...
 
-### NATS Configuration
+### Eventing Configuration
 
 | Parameter | Description | Default Value |
 |---|---|---|
-| `nats.enable` | Enable NATS | `true` |
-| `nats.url` | The NATS Url | `nats://bmrg-nats:4222` |
-| `nats.cluster` | NATS Streaming Cluster | `bmrg-stan` |
+| `eventing.enables` | Enables eventing as part of the installation. This will install the listener service to accept incoming webhooks and events. Optionally requires NATS | `true` |
```
Comment (tlawrie) [inline bmrg-flow/README.md:133]: I think this would be `eventing.enabled`

File: bmrg-flow/values.yaml [hunk 2]
```diff
@@ -163,9 +162,10 @@ monitoring:
       host: jaeger-dev-agent
       port: 5778
 
-jetstream:
-  enabled: false
-  url: nats://localhost:4222
+# Eventing related values
+eventing:
+  enabled: true
```
Comment (tlawrie) [inline bmrg-flow/values.yaml:167]: We don't by default enable eventing. Can we switch this to `false`

**Review by tlawrie — CHANGES_REQUESTED** [review]
Overall, really great and well done change.

Few minor updates needed.

Also need to update the Charts version and appVersion

File: bmrg-flow/README.md (hunk 1 above)
Comment (corcoja) [reply bmrg-flow/README.md]: Typo. Thanks!

File: bmrg-flow/values.yaml (hunk 2 above)
Comment (corcoja) [reply bmrg-flow/values.yaml]: Fixed.

**Comment by corcoja** [comment]
@tlawrie, addressed those few issues you mentioned. Please review.

**Review by tlawrie — APPROVED** [review]

**Pull request merged**